msteams-platform/messaging-extensions/build-bot-based-message-extension.md

```diff
@@ -95,6 +95,29 @@ You must add the following search parameter details that define the text visible
 
 For more information, see [app manifest schema](~/resources/schema/manifest-schema.md).
 
+## Message extension as plugin for Copilot for Teams meetings
+
+Message extension plugins are supported in Copilot for Teams meetings. Extending bot-based message extension plugins to meetings allows developers to enhance the Copilot for Teams meetings. Copilot can utilize various app capabilities during meetings, such as task modules, app sharing, and more, to improve user engagement and productivity. For example, a plugin can be used in Copilot for Teams meetings to summarize discussions, generate notes, and list tasks. It also extends the benefits of traditional plugins by enabling real-time interaction with external data and custom functionality.
+
+Meetings Copilot is interactive and efficient, it can assist with summarizing discussions, suggesting action items, and providing prompt-less assistance in real-time. Users can ask natural language questions to meeting copilot and get responses from your plugin that are relevant to the meeting context. Plugins are available for users to enable in both Copilot for Microsoft 365 and Copilot for Teams meetings.
+
+:::image type="content" source="../assets/images/Copilot/meeting-copilot-extensibility.png" alt-text="Screenshot shows the plugins flyout menu in Copilot for Teams meetings.":::
+
+| Benefits          | Description                                                                                                            |
+|-------------------|------------------------------------------------------------------------------------------------------------------------|
+| User Empowerment  | Extending Meeting Copilot empowers users with various actions to enhance their meeting experience.                      |
+| UI Commands       | Users can easily access features like opening a task module or sharing an app to the stage.                             |
+| Search Commands   | Efficient data retrieval from app databases is made possible, streamlining the search process.                          |
+
+| Scenarios         | Description                                                                                                            |
+|-------------------|------------------------------------------------------------------------------------------------------------------------|
+| Reactive Commands | Users can directly command Copilot to perform specific actions or provide information.                                  |
+|                   | - Natural Language Prompts: Users can instruct Copilot to carry out search or action commands.                         |
```

Comment:
We dont have action commands support today. Let's aim to remove it from the docs.

Reply:
Updated as Users can instruct Copilot to carry out search commands.